content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -73,7 +73,7 @@ pod for a given direction, the connections allowed in that direction from that p
 what the applicable policies allow. Thus, order of evaluation does not affect the policy result.
 
 For a connection from a source pod to a destination pod to be allowed, both the egress policy on
-the source pod and the ingress policy on the destination pod need to allow the connection. If
+the source pod and the ingress policy on the destination pod need to allow the connection. If any one of the 
 either side does not allow the connection, it will not happen.
```

Comment:
The original text sounds correct to me, and your change seems clearly ungrammatical...

Reply:
yes

Reply:
@dipankardas011 Can you explain why it sounds ungrammatical? What rules of grammar does it violate?

Reply:
let me raise this PR again with the same changes through my personal Github profile since my corporate email id has some restrictions. But better give some reasoning @dipankardas011 for the grammar rule violation

Reply:
> @dipankardas011 Can you explain why it sounds ungrammatical? What rules of grammar does it violate?

My bad I said yes because their seems to be no grammatical error before

Reply:
It is implicit that "If either side does not allow the connection, it will not happen." so what the document should shout out to the reader is " If any one of either side does not allow the connection, it will not happen." so that the reader is mindful that if he/she misses on any one of the sides the connection won't work!